Source/zerochecker.cpp

```diff
@@ -162,6 +162,10 @@ juce::String File::relTimeToString(const juce::RelativeTime& t)
 	return ((prd > -1) ? str.substring(0, prd + 4) : str);
 }
 
+File::File(const juce::File file) {
+    m_file = file;
```

Comment:
Slight nitpick, I would move this constructor above File::calculate(). I would also adhere to the existing formatting conventions by placing the opening bracket on a newline.